scripts/category/naquadah_catalyst.zs

```diff
@@ -0,0 +1,262 @@
+//Nuclearcraft steam = 2:1 compression, IC2 steam = 6:1 compression, Forge steam = 24:1 compression
+//conversion ratio worsens by n^2 for each tier skipped
+//crafting speed/costs buffed to support potential of extreme throughput demands of endgame seteps
+function SteamMixers(grade as int) as void {
+
+    var gradeUp = grade + 1;
+    var steamLiquid as ILiquidStack = game.getLiquid('magic_steam_grade_' ~ grade);
+    var steamLiquidUp as ILiquidStack = game.getLiquid('magic_steam_grade_' ~ gradeUp);
+    var cataIn as IIngredient = getIngredientFromString('contenttweaker:naquadah_catalyst_grade_' ~ grade);
+    var cataUp as IIngredient = getIngredientFromString('contenttweaker:naquadah_catalyst_grade_' ~ gradeUp);
+    var steamOutNuc as int = 1000/(grade*grade);
+    var steamOutIC2 as int = 500/(grade*grade);
+    var steamOutForge as int = 125/(grade*grade);
+
+    //====================
+    //Conflux 1
+    //====================
+
+    //IE mixer recipes
+    mods.immersiveengineering.Mixer.addRecipe(steamLiquid * steamOutNuc, /*output*/ <fluid:high_pressure_steam> * 2000, /*input*/ [cataIn], 128);
+    mods.immersiveengineering.Mixer.addRecipe(steamLiquid * steamOutIC2, <fluid:ic2superheated_steam> * 2000, [cataIn], 128);
+    mods.immersiveengineering.Mixer.addRecipe(steamLiquid * steamOutForge, <fluid:steam> * 2000, [cataIn], 128);
+    mods.immersiveengineering.Mixer.addRecipe(steamLiquidUp * 1000, steamLiquid * 2000, [cataUp], 128);
+
+    //IC2 canner recipes
+    mods.ic2.Canner.addEnrichRecipe(steamLiquid * steamOutNuc, /*output*/ <fluid:high_pressure_steam> * 2000, /*input*/cataIn);
+    mods.ic2.Canner.addEnrichRecipe(steamLiquid * steamOutIC2, <fluid:ic2superheated_steam> * 2000, cataIn);
+    mods.ic2.Canner.addEnrichRecipe(steamLiquid * steamOutForge, <fluid:steam> * 2000, cataIn);
+    mods.ic2.Canner.addEnrichRecipe(steamLiquidUp * 1000, steamLiquid * 2000, cataUp);
+
+    //====================
+    //Conflux 2 (2x steam per operation)
+    //====================
+
+    steamOutNuc *= 2;
+    steamOutIC2 *= 2;
+    steamOutForge *= 2;
+
+    //nuclearcraft enricher recipes
+    mods.nuclearcraft.Enricher.addRecipe(cataIn, <fluid:high_pressure_steam> * 4000, /*input*/ steamLiquid * steamOutNuc, /*output*/ 0.5, 0.1, 0.0);
+    mods.nuclearcraft.Enricher.addRecipe(cataIn, <fluid:ic2superheated_steam> * 4000, steamLiquid * steamOutIC2, 0.5, 0.1, 0.0);
+    mods.nuclearcraft.Enricher.addRecipe(cataIn, <fluid:steam> * 4000, steamLiquid * steamOutForge, 0.5, 0.1, 0.0);
+    mods.nuclearcraft.Enricher.addRecipe(cataUp, steamLiquid * 4000, steamLiquidUp * 2000, 0.5, 0.1, 0.0);
+
+    //====================
+    //Conflux 3 (4x steam per operation)
+    //====================
+
+    steamOutNuc *= 2;
+    steamOutIC2 *= 2;
+    steamOutForge *= 2;
+
+    //EIO vat recipes

```

Comment:
If you gonna add `The Vat` recipes, please add them stright into respective .xml files instead of .zs file. This may sounds inconvenient, but sadly, EnderIO machines are not fully integrated into ZenScript, so usually manual .xml editing required. I have some EiO methods left in code, but perfectly there should be none.